src/components/types.ts

```diff
@@ -0,0 +1,3 @@
+export type Variant = 'primary' | 'info' | 'success' | 'warning' | 'danger';
```

Comment:
There should probably be exported from the public API, no? Even though being string unions, still might be useful to be able to reference those for typing stuff perhaps.

Reply:
Also "Variant" and "WeekDays" might be a bit too generic/common/in danger of collisions.
Will get back on that with suggestions

Reply:
`WeekDays` works better being generic. I wouldn't change it. It is just what it says and can be used separately from the calendar if someone just needs such an enum.

Reply:
I was thinking `CalendarWeekDay` but honestly `WeekDays` (plural) works just as well. From what I see most .NET stuff already in use are using `WeekDay` struct which should not conflict, so we can keep it and it was an existing type already.

As for the variant - either `ColorVariant` since it's both chip & badge, or going of the description in the docs https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/inputs/chip#variants
>  chip supports several pre-defined stylistic variants

perhaps `StyleVariant` is more like it and it's reasonably unique and should be semantically correct.